### PR TITLE
add friends app

### PR DESCRIPTION
**Pull request by frostwolm**
### friends app

[Demo](https://frostwolm.github.io/friends-app/) |
[Code base](https://github.com/frostwolm/friends-app)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/frostwolm/friends-app/js/app.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    cardElement.appendChild(photoElement);
+    let ageEelement = document.createElement(`div`);
+    ageEelement.textContent = `${this.friendObj.dob.age} years old`;
+    cardElement.appendChild(ageEelement);
+    let phoneElement = document.createElement(`div`);
+    phoneElement.textContent = this.friendObj.phone;
+    phoneElement.classList.add(`phonetext`);
+    cardElement.appendChild(phoneElement);
+    return cardElement;
+  }
```
[the remark this answers is not in the document]
Comment (frostwolm) [reply submissions/frostwolm/friends-app/js/app.js]: Does it repaints even if element (e.g. cardElement) is not yet append to another Element in DOM?

**Comment by IgorKurkov** [comment]
@frostwolm can you take a look at the comments and give feedback? If you are stuck - feel free to ask :)

**Comment by frostwolm** [comment]
> @frostwolm can you take a look at the comments and give feedback? If you are stuck - feel free to ask :)
Ok. Thank you for your concern. I didn't stuck. I just have no time (
I think will add my commit this weekend.

**Comment by frostwolm** [comment]
Add some fixes and improves. Please review again :)

**Pull request merged**